plugins/GSdx/GSCrc.cpp

```diff
@@ -180,6 +180,8 @@ CRC::Game CRC::m_games[] =
 	{0x6BA2F6B9, ResidentEvil4, EU, 0},
 	{0x60FA8C69, ResidentEvil4, JP, 0},
 	{0x5F254B7C, ResidentEvil4, KO, 0},
+	{0x7C48D927, ResidentEvilCodeVeronicaX, RU, 0}, // unofficial RU-version
+	{0x546C846B, RuleOfRose, RU, 0}, // unofficial RU-version
```

Comment:
These two shouldn't be added. There isn't any code that looks up those specific games.